test/unit/cpu/CMakeLists.txt

```diff
@@ -49,12 +49,12 @@ if (RAJA_ENABLE_AGENCY)
   include_directories(SYSTEM "${PROJECT_SOURCE_DIR}/extra/agency")
 endif()
 
-raja_add_library(NAME bis SOURCES buildIndexSet.cxx)
+#raja_add_library(NAME bis SOURCES buildIndexSet.cxx)
```

Comment:
For now I'm excluding this; because both buildIndexSet.hxx and all of the tests include RAJA.hxx they also include all of the Agency code, which apparently causes multiple-definition errors. I haven't tracked down what about agency/cuda.hpp actually causes this error, but for the sake of testing the code I've excluded bis and hacked around it (see below).